src/python/director/drakevisualizer.py

```diff
@@ -377,6 +381,51 @@ def onViewerDraw(self, msg):
 
         self.view.render()
 
+    def onPlanarLidar(self, msg, channel):
+
+        linkName = channel.replace('DRAKE_PLANAR_LIDAR_', '', 1)
+        robotNum, linkName = linkName.split('_', 1)
+        robotNum = int(robotNum)
+
+        try:
+            link = self.getLink(robotNum, linkName)
+        except KeyError:
+            if linkName not in self.linkWarnings:
+                print 'Error locating link name:', linkName
+                self.linkWarnings.add(linkName)
+        else:
+            if len(link.geometry):
+                polyData = link.geometry[0].polyDataItem
+            else:
+                polyData = vtk.vtkPolyData()
+
+                name = linkName + ' geometry data'
+                geom = type('', (), {})
+                geom.color = [1.0,0.0,0.0,1.0]
+                g = Geometry(name, geom, polyData, link.transform)
+                link.geometry.append(g)
+
+                linkFolder = self.getLinkFolder(robotNum, linkName)
+                self.addLinkGeometry(g, linkName, linkFolder)
+                g.polyDataItem.getChildFrame().copyFrame(link.transform)
+
+            points = vtk.vtkPoints()
+            verts = vtk.vtkCellArray()
+
+            t = msg.rad0
+            for r in msg.ranges:
```

Comment:
this loop could be vectorized with numpy, but I think it's probably fine for now and this will be could be moved to c++ eventually